packages/sdk-ts/src/client/indexer/grpc/IndexerGrpcMegaVaultApi.ts

```diff
@@ -0,0 +1,328 @@
+import { InjectiveMegaVaultRpc } from '@injectivelabs/indexer-proto-ts'
+import BaseGrpcConsumer from '../../base/BaseIndexerGrpcConsumer.js'
+import { IndexerModule } from '../types/index.js'
+import { IndexerGrpcMegaVaultTransformer } from '../transformers/index.js'
+import {
+  UnspecifiedErrorCode,
+  GrpcUnaryRequestException,
+} from '@injectivelabs/exceptions'
```

Comment:
can be: (`types` always at below)
```suggestion
import { InjectiveMegaVaultRpc } from '@injectivelabs/indexer-proto-ts'
import { UnspecifiedErrorCode, GrpcUnaryRequestException } from '@injectivelabs/exceptions'
import BaseGrpcConsumer from '../../base/BaseIndexerGrpcConsumer.js'
import { IndexerGrpcMegaVaultTransformer } from '../transformers/index.js'
import { IndexerModule } from '../types/index.js'
```